src/Components/NavBar/helper-functions.js

```diff
@@ -0,0 +1,19 @@
+export const locationFunction = (location) => {
+    switch(location.pathname) {
+        case '/':
+            return {
+                homeButton: 'hide',
```

Comment:
So innovative to use an object with properties to describe the navigation status